build_projects/dotnet-cli-build/dotnet-cli-build.csproj

```diff
@@ -66,7 +61,6 @@
   </PropertyGroup>
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
```

Comment:
@livarcocc Actually I know what you mean. I see the desired project file in issue #4330